WordPressKit/EditorSettings.swift

```diff
@@ -0,0 +1,27 @@
+private struct RemoteEditorSettings: Codable {
+    let editor_mobile: String
+    let editor_web: String
+}
+
+public enum EditorSettings: String {
+    case gutenberg
+    case aztec
+
+    static let `default` = EditorSettings.aztec
+}
+
+extension EditorSettings {
+    init (with response: AnyObject) throws {
```

Comment:
The space after the init looks off. @wordpress-mobile/platform-9 we don't have swiftlint running on all repos?

Reply:
Nice catch! Fixed 👍

Reply:
Yes, we don't have swiftlint running in the non-app repos. Feel free to open a request for us to add that 😄